app/models/hyrax/batch_ingest/batch_item.rb

```diff
@@ -2,12 +2,26 @@
 
 module Hyrax::BatchIngest
   class BatchItem < ApplicationRecord
-    STATUSES = ['initialized', 'enqueued', 'running', 'completed', 'failed', 'expunged'].freeze
+    STATUSES = ['initialized', 'enqueued', 'running', 'completed', 'failed'].freeze
 
     belongs_to :batch
     validates :status, inclusion: { in: STATUSES }
     paginates_per 20
 
     delegate :submitter_email, :submitter, to: :batch
+
+    def repo_object_gone?
+      status == 'completed' && repo_object_id.nil?
+    end
+
+    def fetch_repo_object
```

Comment:
These two methods look useful but unused by this PR.  Could you either remove them or add tests for them?  If you keep them, maybe `fetch_repo_object` would be best `private`?